web/packages/teleport/src/Roles/RoleEditor/StandardEditor/validation.ts

```diff
@@ -330,15 +342,15 @@ const validKubernetesKind = (
       };
 
     case RoleVersion.V8:
-      const v7groups = kubernetesResourceKindV7Groups[kind];
-      const v8valid =
-        !v7groups || (apiGroup !== '*' && !v7groups.groups.includes(apiGroup));
-      return {
-        valid: v8valid,
-        message: v8valid
-          ? undefined
-          : `Kind must use k8s plural name. Did you mean "${v7groups.v8name}"?`,
-      };
+      return validateV8(kind, apiGroup);
+
+    case RoleVersion.V9:
+      const v8valid = validateV8(kind, apiGroup);
+      if (!v8valid.valid) {
+        return v8valid;
+      }
+      // TODO(@creack): Implement validation for v9.
```

Comment:
I'm assuming that you won't backport this directly until the validation is actually implemented, right?